documentation/api/reference/made_with_ov.md

```diff
@@ -0,0 +1,76 @@
+# Made with Openverse
+
+The Openverse API provides access to the world's largest library of
+openly-licensed creative works, containing over 700 million images and audio
+tracks. All Openverse content is under a
+[Creative Commons license](https://creativecommons.org/licenses/) or is in the
+public domain.
+
+As a developer, you can use the Openverse API to create incredible apps. Here
+are some apps powered by the Openverse API.
+
+```{note}
+If you've made something using the Openverse API that you would like to share,
+[let us know](https://github.com/WordPress/openverse#keep-in-touch) and we'll
+feature it here!
```

Comment:
Maybe link to this file on GitHub so that the users can edit it and open a PR?

Reply:
There is an Edit button on every docs page, in the top-right that leads directly to GitHub's web editor UI.